python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -269,6 +271,52 @@ def test_torch_tensor_nccl(ray_start_regular):
     # ray.get(receiver.ping.remote())
 
 
+@pytest.mark.parametrize("ray_start_regular", [{"num_cpus": 4}], indirect=True)
+def test_torch_tensor_nccl_overlap(ray_start_regular):
```

Comment:
can you add test cases where exception is raised from compute/recv/send? (and it is raised properly)

Reply:
The behavior for exceptions should be the same as previously. Added to `test_torch_tensor_exceptions()`